tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "strictNullChecks": true
   },
   "include": ["src"],
-  "exclude": ["**/*.test?.*"]
+  "exclude": ["**/*.test?.*", "src/snapshot/**/*"]
```

Comment:
I know we've talked about wanting this to be buildable/type-checkable, but I'm not sure how to get this into that sort of state: namely, the generated snapshots have a dependency on @tanstack/react-query and the generated relative imports aren't matching up with what is expected.